app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Elf(Player, ABC):
```

Comment:
And here too

Reply:
i saw that in checklist and decided to do that